sacred/observers/file_storage.py

```diff
@@ -53,7 +55,15 @@ def __init__(
             template = basedir / "template.html"
             if not template.exists():
                 template = None
-        self.initialize(basedir, resource_dir, source_dir, template, priority)
+        self.initialize(
+            basedir,
+            resource_dir,
+            source_dir,
+            template,
+            priority,
+            no_duplicate,
+            should_save_sources,
```

Comment:
I would prefer a name like `save_sources` without should

Reply:
That was my original choice too, but there's already a method of the same name. I can rename the method / happy to use any other name you might prefer?

Reply:
Have now renamed to `copy_sources` in line with `copy_artifacts`.